13LessonThirteen/homework/css/style.css

```diff
@@ -0,0 +1,100 @@
+header {
+  height: 500px;
+  background-image: url("http://bone.themanoid.com/images/office/03.jpg");
+  text-align: center;
+}
+h1 {
+  line-height: 500px;
+  font-size: 5em;
+  font-family: Work Sans,Helvetica Neue,sans-serif;
+  font-weight: 500;
+}
+.bg-format {
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+#man {
+  background-image: url("http://bone.themanoid.com/images/thumbs/03.png");
+}
+#hills {
+  background-image: url("http://bone.themanoid.com/images/thumbs/07.png");
+}
+#devices {
+  background-image: url("http://bone.themanoid.com/images/thumbs/02.png");
+}
+#rockface {
+  background-image: url("http://bone.themanoid.com/images/thumbs/04.png");
+}
+#building {
```

Comment:
FYI the dimensions of these images is inconsistent when the browser is sized to medium screen width - they should all be the same height and width